seahorn/jobs/byte_cursor_compare_lexical/aws_byte_cursor_compare_lexical_harness.c

```diff
@@ -15,13 +15,13 @@ int main() {
 
     /* assumptions */
     assume(aws_byte_cursor_is_valid(&lhs));
-    assume(lhs.ptr != NULL);
+    if (lhs.ptr == NULL) return 0;
     if (nd_bool()) {
         rhs = lhs;
     } else {
         initialize_byte_cursor(&rhs);
         assume(aws_byte_cursor_is_valid(&rhs));
-        assume(rhs.ptr != NULL);
+        if (rhs.ptr == NULL) return 0;
```

Comment:
not part of KLEE related commit.

Why are you changing assumes to something else?